scripts/src/main/resources/scripts/command/aws

```diff
@@ -16,8 +16,22 @@ source "$(dirname "${0}")"/../functions
 
 # Call aws with specified arguments.
 function doRun() {
-  doSetup silent
```

Comment:
why is this not wroking?
This is the common way to do it in all other commandlets.
Cant we make it work the same way here?